internal/usecase/interfaces.go

```diff
@@ -2,8 +2,8 @@
 package usecase
 
 import (
+	"autoDiagnosticService/internal/entity"
 	"context"
-	"github.com/evrone/go-clean-template/internal/entity"
 )
 
 //go:generate mockgen -source=interfaces.go -destination=./mocks_test.go -package=usecase_test
```

Comment:
либо так
```suggestion
//go:generate mockgen -source=interfaces.go -destination=./mocks.go -package=usecase
```

либо так

```suggestion
//go:generate mockgen -source=interfaces.go -destination=./mocks/auth.go -package=mocks
```